src/SDKs/SqlManagement/Sql.Tests/Utilities/SqlManagementTestUtilities.cs

```diff
@@ -40,6 +40,8 @@ public class SqlManagementTestUtilities
 
         public const string DefaultEuapPrimaryLocation = "East US 2 EUAP";
 
+        public const string DefaultEuapPrimaryLocationId = "eastus2euap";
+
         public const string DefaultLogin = "dummylogin";
```

Comment:
I have opened an issue for this - https://github.com/Azure/azure-sdk-for-net/issues/3492 Lets track this separately?